lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
That would be nice, with a conditional import, so if someone is providing bluebird in it's own project, that would use the external one. What do you think? I could very well do thé PR for that as it's more for me than you.